src/gui/mainwindow.cpp

```diff
@@ -1132,10 +1132,12 @@ void MainWindow::showEvent(QShowEvent *e)
 {
     qDebug("** Show Event **");
 
+    e->accept();
+
     if (currentTabWidget() == m_transferListWidget)
         m_propertiesWidget->loadDynamicData();
 
-    e->accept();
+    show();
```

Comment:
Looks weird...
Can you elaborate what happens?
Doesn't it cause `showEvent()` to be invoked again?

Reply:
I pushed another variation (tested locally) and added some comments in the commit.